onnxmltools/convert/common/_apply_operation.py

```diff
@@ -420,6 +420,39 @@ def apply_parametric_softplus(scope, input_name, output_name, container, operato
         # g = a * f
         apply_mul(scope, [fName, aName], output_name, container)
 
+
+def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=None, alpha=[1.0], beta=[1.0]):
```

Comment:
Please don't do `alpha=[1.0]`. Use `None` instead. Pycharm can tell why.

Reply:
Thanks for the catch!